src/SFA.DAS.EmployerIncentives.Domain/ValueObjects/Incentive.cs

```diff
@@ -73,30 +71,33 @@ private static int AgeAtStartOfCourse(DateTime dateOfBirth, DateTime startDate)
             return dateOfBirth.AgeOnThisDay(startDate);
         }
 
-        protected List<Payment> Generate(IEnumerable<PaymentProfile> paymentProfiles, int breakInLearningDayCount)
+        protected List<Payment> Generate(IEnumerable<PaymentProfile> paymentProfiles, IReadOnlyCollection<BreakInLearning> breaksInLearning)
         {
             var payments = new List<Payment>();
-
-            if (!IsEligible)
-            {
-                return payments;
-            }          
+            if (!IsEligible) return payments;
 
             var paymentIndex = 0;
             foreach (var paymentProfile in paymentProfiles)
             {
-                payments.Add(new Payment(paymentProfile.AmountPayable, StartDate.AddDays(paymentProfile.DaysAfterApprenticeshipStart).AddDays(breakInLearningDayCount), _earningTypes[paymentIndex]));
+                payments.Add(new Payment(paymentProfile.AmountPayable, StartDate.AddDays(paymentProfile.DaysAfterApprenticeshipStart), _earningTypes[paymentIndex]));
                 paymentIndex++;
             }
 
+            foreach (var breakInLearning in breaksInLearning)
```

Comment:
Can this logic be moved to the Payment object that is applies to?.  i.e pass in the constructor.  Having an AddDays method on a value object (which should be immutable) seems wrong.